application/common/config/main.php

```diff
@@ -106,20 +106,38 @@
                     'logVars' => [], // Disable logging of _SERVER, _POST, etc.
                     'message' => [
                         'to' => $alertsEmail ?? '(disabled)',
-                        'subject' => 'ERROR - ' . $idpName . ' ID Sync [' . $appEnv .']',
+                        'subject' => 'ERROR - ' . $idpName . ' ID Sync [' . YII_ENV .']',
                     ],
                     'baseUrl' => $emailServiceConfig['baseUrl'],
                     'accessToken' => $emailServiceConfig['accessToken'],
                     'assertValidIp' => $emailServiceConfig['assertValidIp'],
                     'validIpRanges' => $emailServiceConfig['validIpRanges'],
-                    'prefix' => function ($message) use ($appEnv, $idpName) {
+                    'prefix' => function ($message) use ($idpName) {
                         return Json::encode([
-                            'app_env' => $appEnv,
+                            'app_env' => YII_ENV,
                             'idp_name' => $idpName,
                         ]);
                     },
                     'exportInterval' => 1,
                 ],
+                [
+                    'class' => SentryTarget::class,
+                    'enabled' => true,
```

Comment:
If desired, you could also make this conditionally enabled, based on whether there is a non-empty `SENTRY_DSN` env. var. value. Just a thought.